services/accounting/controllers/invoice.ts

```diff
@@ -61,12 +65,83 @@ const updateInvoice = asyncHandler(async (req, res, next) => {
   res.json(200);
 });
 
-const createInvoice = asyncHandler(async (req, res, next) => {
-  // TODO - call ZBD api to create an invoice
-  // save the invoice data to the database
-  // return the invoice data to the client
+const createInvoice = asyncHandler(async (req, res: any, next) => {
+  const request = {
+    trackId: req.body.trackId,
+    amount: req.body.amount * 1000, // Convert to msats
+    type: req.body.type ? req.body.type : "boost",
+    metadata: req.body.metadata ? req.body.metadata : null,
```

Comment:
nit - ternary doesnt look necessary here, can just set the value
```suggestion
    metadata: req.body.metadata,
```